app/src/main/java/io/apicurio/registry/storage/impl/jdbc/JdbcEntityMapper.java

```diff
@@ -0,0 +1,18 @@
+package io.apicurio.registry.storage.impl.jdbc;
+
+import io.apicurio.registry.storage.StoredArtifact;
+import io.apicurio.registry.storage.impl.jdbc.entity.Artifact;
+
+import javax.enterprise.context.Dependent;
+
+@Dependent
+public class JdbcEntityMapper {
```

Comment:
Why a separate class?
Is it used elsewhere as well?
If not, I would just move the method into storage impl.

Reply:
I like the separation of concerns, I don't think it's a big issue, but can move it if it becomes one